src/super_gradients/training/transforms/transforms.py

```diff
@@ -441,6 +444,9 @@ def __call__(self, sample: dict) -> dict:
         sample["image"] = (sample["image"] / self.max_value).astype(np.float32)
         return sample
 
+    def get_equivalent_preprocessing(self) -> List[Dict]:
+        return [{Processings.StandardizeImage: {"max_value": self.max_value}}]
```

Comment:
Why don't we directly instantiate it? I know we pass it through the factory, but the factory can also be support instantiated object. Is there another reason?

It would look more simple, more explicit, and easier to debug  + here we cannot directly click on the class StandardizeImage.

Reply:
Since we also need to revert back to "factory processable" input to save them inside the checkpoint file, I just do it first here (also thought it is not a good practice to import these objects here).
It is less risky translating this into instances rather then the other way around.